models/_sector/dex/trades/platforms/uniswap/ethereum/uniswap_ethereum_v3_base_trades.sql

```diff
@@ -0,0 +1,49 @@
+{{ config(
+    tags=['dunesql'],
+    schema = 'uniswap_v3_ethereum',
+    alias = 'base_trades',
```

Comment:
By the way since legacy spells are removed, do we still need the `alias()` here?
e.g.
```suggestion
    alias = alias('base_trades'),
```